### PR TITLE
Fix cni dependency error on upgrade

### PR DESCRIPTION
**Pull request by jakolehm**

### CODE REVIEW

File: lib/pharos/host/ubuntu/scripts/install-kube-packages.sh [hunk 1]
```diff
@@ -4,7 +4,7 @@ set -e
 
 export DEBIAN_FRONTEND=noninteractive
 apt-mark unhold kubectl kubeadm || echo "Nothing to unhold"
-apt-get install -y "kubectl=${KUBE_VERSION}-00" "kubeadm=${KUBEADM_VERSION}-00"
+apt-get install -y "kubectl=${KUBE_VERSION}-00" "kubeadm=${KUBEADM_VERSION}-00" "kubeadm=${KUBE_VERSION}-00"
```
Comment (jnummelin) [inline lib/pharos/host/ubuntu/scripts/install-kube-packages.sh]: this is not right, should be the same as for Debian above

Comment (jakolehm) [reply lib/pharos/host/ubuntu/scripts/install-kube-packages.sh]: yeah, I noticed after I pushed.. I'll push a fix once I have tested upgrade path.

Comment (jakolehm) [reply lib/pharos/host/ubuntu/scripts/install-kube-packages.sh]: Fixed.

**Comment by jakolehm** [comment]
Tested upgrade on Ubuntu 18.04 from 2.3.1 and it seems to work.

**Comment by jakolehm** [comment]
@jnummelin @kke PTAL

**Pull request merged**